### ISSUE TITLE
ArangoDB collection schemas and arango-orm field descriptions

### ISSUE THREAD

**Issue by incorvia** [threatify/arango-orm#111]
It seems to me that when I define fields within a collection it does not translate in any way into an ArangoDB schema which the DB validates future objects against. Is my understanding correct? If so, is there any thought of adding this such that defining fields on a collection will automatically update the schema with ArangoDB?

**Comment by kashifpk** [comment]
Yes currently there is no support of ArangoDB's schema in arango-orm. There has been a discussion about it [here](https://github.com/threatify/arango-orm/issues/97) and [here](https://github.com/threatify/arango-orm/issues/32).

The problem is that arango-orm started using marshmallow before jsonschema was introduced in arango. While using json schema makes a lot more sense now, the switch is not going to be backwards compatible.
